include/ebpf_program_types.h

```diff
@@ -7,12 +7,14 @@
 
 #include <guiddef.h>
 #if !defined(NO_CRT) && !defined(_NO_CRT_STDIO_INLINE)
+#include <stdbool.h>
 #include <stdint.h>
 #else
 typedef unsigned char uint8_t;
 typedef unsigned int uint32_t;
 typedef unsigned long long uint64_t;
 typedef unsigned short wchar_t;
+#define bool _Bool
```

Comment:
I thought if you did line 10 you shouldn't need line 17

Reply:
Dave they are in #if and #else respectively. `stdbool` is good for user mode binaries. Whereas the KM binaries cannot include stdbool.